src/minecraft/commands/WeightCommand.js

```diff
@@ -0,0 +1,70 @@
+const MinecraftCommand = require('../../contracts/MinecraftCommand')
+const fetch = require('node-fetch')
```

Comment:
Maybe worth adding a cache adapter like [Node-fetch-cache](https://www.npmjs.com/package/node-fetch-cache) or [Axios-cache-adapter](https://www.npmjs.com/package/axios-cache-adapter) to cache responses temporarily, so if someone were to run `!weight` and immediately `!skills` for the same username it wouldn't need to be fetched twice.

Reply:
okay done